templates/shadow-cljs.edn

```diff
@@ -1,5 +1,9 @@
 {:source-paths ["src"]
  :dependencies []
+ 
+ :dev-http {5000 "build/browser"}
```

Comment:
Nice! I didn't notice that `shadow-cljs` provided a server for this :)

This however means that now the [`start:browser` target](https://github.com/lambrospetrou/create-shadow-cljs-app/blob/master/templates/package.json#L13) is no longer needed, neither the [`serve`](https://github.com/lambrospetrou/create-shadow-cljs-app/blob/master/templates/package.json#L18) dependency.

The instructions will need to be updated as well since now we don't need to start the server manually.